crates/config/README.md

```diff
@@ -115,6 +115,10 @@ no_match_contract = "Bar"
 match_path = "*/Foo*"
 no_match_path = "*/Bar*"
 no_match_coverage = "Baz"
+# max concurrent threads to use for tests
+max_threads = 16
```

Comment:
I would name this just "threads" with an alias for "jobs"/"j", and remove "test" language from docs since we'll want to use this everywhere

Reply:
changed it in https://github.com/foundry-rs/foundry/pull/8341/commits/f9bc1c9819b4a7112a2e68616f30332ddddbcfef `-j` is already used by json format so I just added short `-t` for it